package.json

```diff
@@ -19,16 +19,23 @@
   "devDependencies": {
     "@playwright/test": "1.19.0",
     "@typescript-eslint/eslint-plugin": "^5.7.0",
-    "@typescript-eslint/parser": "^5.7.0",
+    "@typescript-eslint/parser": "^5.15.0",
```

Comment:
All these dependencies were required by `lint-e2e`, which runs on each commit. I think they should be in devDependencies then, is that correct?

Reply:
Seems like a lot of packages! If we need them then, yes, that is where they should go.